Remove query/filters from elasticSearchFunctions

jraddaoui opened the pull request:
The match all query and backlog filters attributes are prone to be
modified externally causing issues when they're obtained after for other
queries.

Connects to https://github.com/archivematica/Issues/issues/452.

ross-spencer reviewed (approved):
LGTM! :+1: 

I thought this was an interesting issue. Do you think the other values in `elasticsearchfunctions` are likely to be impacted in a similar way? e.g. `BACKLOG_FILTER` 

I notice this isn't building in Python 2.7 but that looks to be because of the recent updates also being discussed in the Archivematica channel.

jraddaoui commented:
The tests are failing due to a dependency installation issue, which I think @sevein is addressing in here alongside https://github.com/artefactual/archivematica-storage-service/pull/437.

jraddaoui commented:
Good point @ross-spencer! The backlog filters are not being modified anywhere, but they're also prone to the same issue in the future and, considering their use, it's a good idea to remove them in here too.

jraddaoui committed:
Remove query/filters from elasticSearchFunctions

The match all query and backlog filters attributes are prone to be
modified externally causing issues when they're obtained after for other
queries.

jraddaoui commented:
Back to you @ross-spencer. Sorry, I had to amend the commit.

ross-spencer reviewed (approved):
¡Perfecto! :+1:
